src/Api/Validator.php

```diff
@@ -249,7 +249,20 @@ private function checkCanString($value)
 
     private function checkAssociativeArray($value)
     {
-        if (!is_array($value) || isset($value[0])) {
+        $assoc = false;
```

Comment:
Style nitpick: can the variable name be more descriptive, like `$isAssociative`?